src/java/org/apache/nutch/api/NutchServer.java

```diff
@@ -85,7 +88,12 @@
    * well as the logging granularity. If the latter option is not provided via
    * {@link org.apache.nutch.api.NutchServer#main(String[])} then it defaults to
    * 'INFO' however best attempts should always be made to specify a logging
-   * level.
+   * level.<br>
+   * {@link org.apache.nutch.api.NutchServer} can be run as secure. restapi.auth property
+   * should be set to true at <code>nutch-site.xml</code> to enable HTTP basic authentication
```

Comment:
In JDK 1.8 I think the relevnt HTML encoding should be used e.g. &lt;code&gt;nutch-site.xml&lt;/code&gt;